platform/src/test/java/platformstepdefinition/PF_Login.java

```diff
@@ -77,28 +78,67 @@ public void navigate_to_home_page() throws Throwable {
 
 		loginToPlatform.navigateToHomePage();
 	}
-	/*
-	 * @And("^Close browser$") public void close_browser() throws Throwable { // //
-	 * closebrowser(); }
-	 */
 
-	/*
-	 * @Given("^Navigate to URL with (.+) and (.+) PFQA_123_1$") public void
-	 * navigate_to_url_with_and_pfqa1231(String config, String environment) throws
-	 * Throwable { // throw new PendingException(); driver =
-	 * openBrowserstack(config, environment); // driver
-	 * 
-	 * }
+	/**
+	 * @Author : Chetan Sonparote
+	 * @Date : 5 Aug 2021
+	 * @Description: Added method for validating login
 	 */
 
-	/*
-	 * @AfterTest public void tearDown() { driver.quit(); }
-	 */
-	/*
-	 * @Given("^Navigate to URL with (.+) PFQA_123_1$") public void
-	 * navigate_to_url_with_pfqa1231(String config) throws Throwable { // throw new
-	 * PendingException(); String environment = null; driver =
-	 * openBrowserstack(config, environment); }
-	 */
+	@When("^User clicks login button$")
+	public void user_clicks_login_button() throws Throwable {
+		// throw new PendingException();
+		LoginPage loginPage = new LoginPage(driver);
+		loginPage.clickLoginButton();
+	}
+
+	@Then("^Validate that error is displayed for invalid user name$")
+	public void validate_that_error_is_displayed_for_invalid_user_name() throws Throwable {
+		// throw new PendingException();;
+		NewBaseClass newBaseClass = new NewBaseClass();
+		LoginPage loginPage = new LoginPage(driver);
+		newBaseClass.validateErrorMessage(loginPage.invalidDataErrorText, "メールアドレスは、メールアドレス形式で入力してください。");
```

Comment:
content should not use. Please use it through excel.

Reply:
Validating error message as done with Order module.
Seperate script to be written for validating strings